DOC-843: RTC docs revisions for 1.0 release (#2091)
tinymce/tinymce-docs: _includes/auth/private-public-key-pairs-for-tiny-cloud-services.md

## Add a public key to the {{site.cloudname}} API key

> The {{pluginname}} Server requires a _public_ key generated from the same _private_ key that will be used on your JSON Web Token (JWT) Provider server. The public key(s) stored on the {{pluginname}} Server are used to ensure that content is sent by authorized users.

Reviewer: This uses different terms to what's used elsewhere which could cause some confusion? e.g here it uses `JSON Web Token (JWT) Provider server` but in how-jwts-are-used.md it uses `JSON Web Token (JWT) endpoint`

Reviewer: We also use `endpoint` in jwt-authentication.md, so whatever we change it to we'll need to make sure that's updated as well.

Author: Fixed by https://github.com/tinymce/tinymce-docs/pull/2091/commits/b3091a1ad3845121811a20d729af9663174b26ec

There are two methods for generating and adding a public key to your API key:

> 1. The secure key pair generator at [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) (recommend).

Reviewer 2: Change "recommend" at the end to "recommended"

Author: oops :facepalm:

Author: Fixed in https://github.com/tinymce/tinymce-docs/pull/2091/commits/0c30d3f05b1fd7a4901fcdcd00832987e118e238

> 1. The secure keypair generator at [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) (recommend).
> 1. Generate a keypair locally and add the _public_ key to [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/).

Reviewer: ```suggestion
1. The secure key pair generator at [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) (recommend).
1. Generate a key pair locally and add the _public_ key to [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/).
```
`keypair` isn't a word and looking at other terms used elsewhere it's always 2 words.

> ### Generate a keypair using the {{site.accountpage}} JWT Keys page

Reviewer: ```suggestion
### Generate a key pair using the {{site.accountpage}} JWT Keys page
```

> The [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) page provides a private/public key generator. This generator will store a copy of the _public_ key, and provide a downloadable file for both the public and private keys. {{site.companyname}} does not store the _private_ key and the key-pair cannot be retrieved later.

Reviewer: ```suggestion
The [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) page provides a private/public key generator. This generator will store a copy of the _public_ key, and provide a downloadable file for both the public and private keys. {{site.companyname}} does not store the _private_ key and the key pair cannot be retrieved later.
```
Here we used a hyphenated version further adding inconsistency?

> ### Generate a keypair locally
> 
> When generating a keypair locally, use one of the supported algorithms.
> {% include auth/jwt-supported-algorithms.md %}
> 
> For instructions on generating a keypair locally, see: [Creating a private/public key pair for Tiny Cloud]({{site.baseurl}}/advanced/generate-rsa-key-pairs/).

Reviewer: ```suggestion
### Generate a key pair locally

When generating a key pair locally, use one of the supported algorithms.
{% include auth/jwt-supported-algorithms.md %}

For instructions on generating a key pair locally, see: [Creating a private/public key pair for Tiny Cloud]({{site.baseurl}}/advanced/generate-rsa-key-pairs/).
```
